docs/models/peering/routingpolicy.md

```diff
@@ -0,0 +1,28 @@
+# Routing Policy
+
+Routing policies are a set of instructions to apply when advertising or
+receiving prefixes from another autonomous system. They are meant to filter
+routes to import or export. Routing policies play an important role to secure
+BGP by rejecting routes that does not belong in the Internet. Policies can in
+most cases be chained to achieve a complex and effective filtering process
+while keeping policy as simple as possible.
```

Comment:
"Routing policies play an important role in securing BGP by rejecting invalid routes. Policies can be be chained to achieve complex and effective filtering, while keeping policies as simple as possible"